controllers/reservations.js

```diff
@@ -16,26 +24,16 @@ router.post('/', function(req, res) {
 // Occupy spot passed in the id
 router.put("/:id/occupy", function(req, res) {
   Reservation.occupy(req.params.id).then(function(reservation) {
-    console.log(reservation);
-    res.json(reservation);
+    res.status(200).json(reservation);
   }).catch(function(error) {
     res.status(500).json({error: error});
   });
 });
 
-// Occupy spot passed in the id
+// Finish spot occupation passed in the id
 router.put("/:id/finish", function(req, res) {
   Reservation.finish(req.params.id).then(function(reservation) {
-    res.json(reservation);
```

Comment:
This looks worse than it is, and it's GitHub being weird with diff. I just moved the 'get reservation' endpoint up to the top of the file.